hydra-node/json-schemas/api.yaml

```
@@ -141,6 +141,22 @@ channels:
           type: request
           method: POST
           bindingVersion: '0.1.0'
+  /protocol-parameters:
+    description: Endpoint to obtain protocol parameters used in hydra-node.
+    servers:
+      - localhost-http
+    subscribe:
+      summary: Get cardano protocol parameters.
+      operationId: getProtocolParameters
+      description: |
+        Clients can request from hydra-node it's protocol parameters used on start.
+      message:
+        $ref: "api.yaml#/components/messages/ProtocolParameters"
```

Should: try to re-use [this json schema](https://github.com/CardanoSolutions/cardanonical/blob/main/cardano.json#L1209)

```suggestion
        $ref: "https://raw.githubusercontent.com/CardanoSolutions/cardanonical/main/cardano.json#/definitions/ProtocolParameters<Babbage>"
```